sr_hand_health_report/src/sr_hand_health_report/overrun_check.py

```diff
@@ -26,58 +26,101 @@ class OverrunCheck(SrHealthReportCheck):
     PASSED_THRESHOLDS = {'overrun_average': 1, 'drop_average': 1}
 
     def __init__(self, hand_side, fingers_to_test):
+        """
+            Initialize the OverrunCheck object
+            @param hand_side: String indicating the side
+            @param fingers_to_test: List of finger prefixes to test
+        """
         super().__init__(hand_side, fingers_to_test)
 
+        self._name = "Overrun"
         self.number_of_drops = 0
         self.iterations = 0
         self.overrun_average = 0
         self.drop_average = 0
-        self._result = None
-
-        rospy.Subscriber("/diagnostics_agg", DiagnosticArray, self.overruns_callback)
-        rospy.Subscriber(f"/{self._hand_prefix}/debug_etherCAT_data", EthercatDebug, self.drops_callback)
-
-    def overruns_callback(self, data):
-        overrun = OverrunCheck.get_recent_overruns(data)
-
-        self.overrun_average += int(float(overrun))
+        self._result = {'overrun': {}}
+
+        rospy.Subscriber("/diagnostics_agg", DiagnosticArray, self._overruns_callback)
+        rospy.Subscriber(f"/{self._hand_prefix}/debug_etherCAT_data", EthercatDebug, self._drops_callback)
+
+    def _overruns_callback(self, data):
+        """
+            Overrun callback
+            @param: DiagnosticArray data
+        """
+        overrun_value = OverrunCheck.get_recent_overruns(data)
+        self.overrun_average += int(float(overrun_value))
         self.drop_average += self.number_of_drops
         self.number_of_drops = 0
         self.iterations += 1
 
-    def drops_callback(self, data):
+    def _drops_callback(self, data):
```

Comment:
The name is a bit misleading IMHO. I thought the method was dropping (i.e. pausing) a callback. Could you rename or simply change the pydoc to be a bit more self-explanatory, please?